charts/k8s-service/values.yaml

```diff
@@ -142,6 +142,22 @@ shutdownDelay: 5
 #         value: docker
 sideCarContainers: {}
 
+# canary specifies test pod(s) that are deployed alongside your application's stable track pods.
+# It is useful for testing a new release candidate in a production environment with minimal disruption and
+# for allowing you to find any issues early.
+#
+# The following example specifies a simple canary deployment:
+#
+# EXAMPLE:
+#
+# canary:
+#   enabled: true
+#     replicaCount: 1
```

Comment:
Indentation looks off here. It should be aligned with `enabled`.

Reply:
Fixed in d1aa186